citylistmetadata/src/types.ts

```diff
@@ -0,0 +1,35 @@
+import { parseXml } from 'libxmljs'
+
+export interface Adresa {
+    ulice: string | null
+    cisloDomovni: string | null
+    cisloOrientacni: string | null
+    obec: string | null
+    obecKod: string | null
+    PSC: string | null
+    castObce: string | null
+    kraj: string | null
+    adresniBod: string | null
+}
+
+export interface PravniForma {
+    type: number
+    label: string
+}
+
+export interface Subjekt {
+    zkratka: string
+    ICO: string | null
```

Comment:
Daval bych prednost vzdy zacinat malym pismenem, i kdyby to byla zkratka. Tedy ICO vs ico a PSC vs psc. Vetsina parseru a generatoru kodu to automaticky hodi na iCO a pSC, coz vypada hrozne. Viz muj generovany kod v [City.kt](https://github.com/cityvizor/cityvizor/blob/master/server-kotlin/src/city_search/City.kt)